Dockerfile

```diff
@@ -25,12 +34,13 @@ ADD build-config.json /opt/opentripplanner/build/
 ADD otp-config.json /opt/opentripplanner/build/
 ADD $OSM_PBF_URL /opt/opentripplanner/build/
 ADD $GTFS_URL /opt/opentripplanner/build/gtfs.zip
+RUN cp mfdz.bb.gtfs.zip /opt/opentripplanner/build/gtfs-carpool.zip
```

Comment:
```suggestion
# GTFS Daten von fahrgemeinschaft mifaz
RUN --mount=type=secret,id=GTFS_CARPOOL_URL curl -L $(cat /run/secrets/GTFS_CARPOOL_URL) -o /opt/opentripplanner/build/gtfs-carpool.zip
```

If we move this down here (because it changes more often than `otp-config.json`, etc.), above the respective `RUN` instruction must be removed.

Reply:
Thanks for the improvement